components/VAINA/src/Vaina.cpp

```diff
@@ -0,0 +1,196 @@
+/**
+ * @file Vaina.cpp
+ * @author Locha Mesh Developers (contact@locha.io)
+ * @brief VAINA - Versatile Address Interface | Network Administration
+ * @version 0.1
+ * @date 2020-04-07
```

Comment:
Current date. Also, @luisan00 why we don't remove this from source files and adopt a more general headers for source files? Something between the lines of:

```
/**
 * Copyright 2020 btcven and Locha Mesh developers
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

This way we don't have to mantain versions numbers and dates. Also we could add the doxygen comment with the author, and a brief description of what the file contains.

Reply:
Ok, it can work, adding below that heading a short description in two or three lines or so.
This may be in another PR for to change all the headers to the new format

Reply:
@jeandudey,  it may be something like:
```cpp
/**
 * Copyright 2020 btcven and Locha Mesh developers
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
/**
 * @Brief:
 *        This code can fry your computer
 */
```

Reply:
Agreed, better for a new PR.